types/index.d.ts

```diff
@@ -0,0 +1,80 @@
+// Type definitions for Chalk 2
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export = chalk;
+
+declare function chalk(...text: (string | number)[]): string;
+declare function chalk(text: TemplateStringsArray, ...placeholders: (string | number)[]): string;
+
+declare namespace chalk {
+
+	interface ChalkConstructorOptions {
+		enabled?: boolean;
+		level?: number;
+	}
+
+	export function constructor(options?: ChalkConstructorOptions): typeof chalk;
+
+	export let enabled: boolean;
+	export let level: number;
+	export const supportsColor: boolean;
```

Comment:
Go to definition always brings you to the type definition file and not the actual implementation so I don't know how helpful it is.
But it would be possible to create a definition for `supports-color` and add it as a dependency.